cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/vagrant/nitro-NITRO-2.7/build/.waf-1.7.8-77e2cdd38ea2930f982e3aabf335fdb7/waflib/Tools/c_config.py", line 460, in run_c_code\n    bld.compile()\n  File "/vagrant/nitro-NITRO-2.7/build/.waf-1.7.8-77e2cdd38ea2930f982e3aabf335fdb7/waflib/Build.py", line 198, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 22224848: cprogram test.c.1.o -> testprog}\n[\'/usr/bin/gcc\', \'test.c.1.o\', \'-o\', \'/vagrant/nitro-NITRO-2.7/target/.conf_check_3803bcf5f374cc3775fd9505c1716854/testbuild/testprog\', \'-Wl,-Bstatic\', \'-Wl,-Bdynamic\', \'-lsqrt\']\n'
